xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecAndroidMediaCodec.cpp

```diff
@@ -493,8 +493,16 @@ bool CDVDVideoCodecAndroidMediaCodec::Open(CDVDStreamInfo &hints, CDVDCodecOptio
       }
       break;
     case AV_CODEC_ID_HEVC:
-      m_mime = "video/hevc";
-      m_formatname = "amc-h265";
+      if (m_hints.codec_tag == MKTAG('d', 'v', 'h', 'e'))
+      {
+        m_mime = "video/dolby-vision";
+        m_formatname = "amc-dvhe";
+      }
+      else
+      {
+        m_mime = "video/hevc";
+        m_formatname = "amc-hevc";
```

Comment:
is the change from `amc-h265` to `amc-hevc` intentional?

Reply:
https://developer.android.com/reference/android/media/MediaCodec#createDecoderByType(java.lang.String) <- Choose what you like :-)

Reply:
m_formatname is only for display in Kodi codec info OSD, IMO hevc is more common then h265